src/components/cms/BotBuilder/BotWizard.js

```diff
@@ -540,20 +530,6 @@ class BotWizard extends React.Component {
     }
   };
 
-  check = () => {
```

Comment:
How is this check handled now?

Reply:
The check will no longer be required, earlier it was used to check whether the user has passed the `Configure` tab before deploying. Now user will be allowed to go to `Deploy` through it, anyways I have added the check in `handleNext`